testing/test_aux.sh

```diff
@@ -18,6 +18,9 @@ echo Lint checks | tee -a $TEST_RESULTS
 bin/check_style
 echo check_style exit code $? | tee -a $TEST_RESULTS
 
+# Remove report file for faux-1 device
```

Comment:
Why?  Something doesn't seem right here.

Reply:
I've decided to copy the faux-1 report from the inst folder to the out folder so it's easier to find for diff-ing and deciding to move it to the docs folder. It's a bit hidden in the inst/report folder.
So it needs erasing in case a test run has been done before to avoid confusion.